pkgs/development/python-modules/pyopencl/default.nix

```diff
@@ -11,16 +11,18 @@
 , six
 , opencl-headers
 , ocl-icd
+, pybind11
+, pkgs
 }:
 
 buildPythonPackage rec {
   pname = "pyopencl";
   version = "2018.2.2";
 
   checkInputs = [ pytest ];
-  buildInputs = [ opencl-headers ocl-icd ];
+  buildInputs = [ opencl-headers ocl-icd pkgs.pybind11 ];
 
-  propagatedBuildInputs = [ numpy cffi pytools decorator appdirs six Mako ];
+  propagatedBuildInputs = [ numpy cffi pytools decorator appdirs six Mako pybind11 ];
```

Comment:
The Python `pybind11` should go into `nativeBuildInputs`

Reply:
No, wait, this is non-native because it will be linked to, so `buildInputs`. @dotlambda seems we have a case here where `setup_requires` does *not* map to `nativeBuildInputs`.

Reply:
But shouldn't that linking be covered by `pkgs.pybind11`? Maybe it's our pybind11 packaging that needs fixing.

Reply:
The Python version of the package essentially has the same data, the headers. We have to use the Python version because it's added in `setup_requires`, but other then that, we should be able to use any of the two in `buildInputs`.

Reply:
I see. How about we just patch it out of `setup_requires`?

Reply:
@Ma27 `pybind11` should be moved to `buildInputs`, shouldn't it?

Reply:
you're right. Updated the patch accordingly.